src/Nancy.Demo.Hosting.Aspnet/Views/razor-strong.vbhtml

```diff
@@ -1,11 +0,0 @@
-﻿@ModelType Nancy.Demo.Hosting.Aspnet.Models.RatPack

```

Comment:
Not sure why this file was removed? Must have been unintentional. There is still a route that attempt to render it https://github.com/phillip-haydon/Nancy/blob/render-razor-app_code_helpers/src/Nancy.Demo.Hosting.Aspnet/MainModule.cs#L124